cypress/integration/e2e/oneMoreTests.js

```diff
@@ -0,0 +1,35 @@
+describe('Actions', () => {
+    before(() => {
+        cy.visit('/')
+    });
+
+    describe('given `Conjugation` page', () => {
+        before(() => {
+            cy.visit('verbs/conjugation/')
+        });
+        describe('when user selects `Past Perfect Conjugation` in drop down and clicks `Start` button', () => {
+            before(() => {
+                cy.get('.choosebox').select('Past Perfect Conjugation');
```

Comment:
we should use the following priorities for selectors:
Selenium best practices mentions order of preference: id > name > css > xpath

So please exchange your selector `cy.get('.choosebox')` to `cy.get('[name="tense_id"]')`